starter-project/src/app/api/user/accounts/route.tsx

```diff
@@ -0,0 +1,31 @@
+import { NextResponse } from 'next/server';
+import { Bruinen } from '@bruinenco/bruinen-sdk';
+const _ = require('lodash');
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const userId = searchParams.get('userId');
+
+  if (!userId) {
+    return new Response('User ID not found', { status: 400 });
+  };
+
+  // Instantiate the Bruinen SDK: https://docs.bruinen.co/api-reference/bruinen-sdk
+  const bruinen = new Bruinen({
```

Comment:
I think this can be done outside of the function so it is only instantiated once.